main.go

```diff
@@ -199,7 +200,7 @@ func (httpConf httpConfigurations) readMessage(msg queueConsumer.Message) {
 	err := sendToWriter(ingestionType, strings.NewReader(msg.Body), uuid, httpConf.baseURLSlice)
 
 	if err != nil {
-		log.Errorf("Error processing msg: %v with error %v", msg, err)
+		log.Errorf("%v", err)
```

Comment:
Should we log the uuid separately so that we can easily extract it and republish?